addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -106,7 +106,20 @@ export const uploadService = {
                 // limited by bandwidth.
                 for (const sortedFile of sortedFiles) {
                     const file = progressToast.files[sortedFile.progressToastId];
-                    const dataURL = await getDataURLFromFile(sortedFile);
+                    let dataURL;
+                    try {
+                        dataURL = await getDataURLFromFile(sortedFile);
+                    } catch {
+                        deleteFile(file.id);
+                        env.services.notification.add(
+                            _.str.sprintf(
+                                env._t('Could not load the file "%s".'),
```

Comment:
Could you add this line to the `.pot` file since it will need new translations ?

Also, could you use `_t` from `odoo/addons/web/static/src/core/l10n/translation.js` instead of the one from the environment ? (will be easier on the forward ports)

Reply:
@abd-msyukyu-odoo Done changes :+1: